### PR TITLE
Global settings

### PR DESCRIPTION
**Pull request by jorenham**
To add a setting, edit `src/resources/settings/lifetiles.conf` and add it as `key = value`. To obtain it use `Settings.get("key")` or `Settings.getBoolean("key")` if it is a parse-able to a boolean.

Deadline: 23:59 01-06-2015

### CODE REVIEW

File: src/main/java/nl/tudelft/lifetiles/graph/model/GraphContainer.java [hunk 1]
```diff
@@ -55,8 +66,12 @@ public GraphContainer(final Graph<SequenceSegment> graph) {
      */
     private void alignGraph() {
         UnifiedPositionTraverser upt = new UnifiedPositionTraverser();
-        EmptySegmentTraverser est = new EmptySegmentTraverser();
-        graph = est.addEmptySegmentsGraph(upt.unifyGraph(graph));
+        if (Settings.getBoolean(SETTING_EMPTY)) {
+            EmptySegmentTraverser est = new EmptySegmentTraverser();
+            graph = est.addEmptySegmentsGraph(upt.unifyGraph(graph));
```
Comment (joswinter) [inline src/main/java/nl/tudelft/lifetiles/graph/model/GraphContainer.java]: `upt.unifyGraph(graph)` is double code. Better use this structure: `new ReferencePositionTraverser(reference).referenceMapGraph(graph);`. The `graph =` isn't needed.

**Comment by albert-smit** [comment]
looks good

**Comment by alangerak** [comment]
Looks good

**Comment by rutgervandenberg** [comment]
Looks good.

**Pull request merged**